config/crd/bases/v1/datadoghq.com_datadogautoscalings.yaml

```diff
@@ -0,0 +1,240 @@
+                - targetRef
+              type: object
+            status:
+              description: DatadogAutoscalingStatus defines the observed state of DatadogAutoscaling
+              properties:
+                conditions:
+                  description: Conditions describe the current state of the DatadogAutoscaling operations.
+                  items:
+                    description: DatadogAutoscalingCondition describes the state of DatadogAutoscaling.
+                    properties:
+                      lastTransitionTime:
+                        description: Last time the condition transitioned from one status to another.
+                        format: date-time
+                        type: string
+                      message:
+                        description: A human readable message indicating details about the transition.
+                        type: string
+                      reason:
+                        description: The reason for the condition's last transition.
+                        type: string
+                      status:
+                        description: Status of the condition, one of True, False, Unknown.
+                        type: string
+                      type:
+                        description: Type of DatadogMetric condition.
+                        type: string
+                    required:
+                      - status
+                      - type
+                    type: object
+                  type: array
+                  x-kubernetes-list-map-keys:
+                    - type
+                  x-kubernetes-list-type: map
+                horizontal:
+                  description: Horizontal is the status of the horizontal scaling, if activated.
+                  properties:
+                    currentReplicas:
+                      description: CurrentReplicas is the current number of replicas for the resource
+                      format: int32
+                      type: integer
+                    desiredReplicas:
+                      description: DesiredReplicas is the desired number of replicas for the resource
+                      format: int32
+                      type: integer
+                    updateTime:
+                      description: UpdateTime is the timestamp from last recommendation
+                      format: date-time
+                      type: string
+                  required:
+                    - currentReplicas
+                    - desiredReplicas
+                    - updateTime
+                  type: object
+                vertical:
+                  description: Vertical is the status of the vertical scaling, if activated.
+                  properties:
+                    desiredResources:
+                      description: DesiredResources is the desired resources for containers
+                      items:
+                        properties:
+                          limits:
```

Comment:
same comment as above for the type of resource this applies to

Reply:
YAML is generated so discarding this comment and keeping one in code.